base_classes/NXdata.nxdl.xml

```diff
@@ -398,21 +398,43 @@
 	</field>
 
 	<!-- Data vs. plot coordinates -->
-	<field name="scaling_factor" type="NX_FLOAT">
+	<field name="FIELDNAME_scaling_factor" type="NX_FLOAT">
 		<doc>
-			The elements in data are usually float values really. For
-			efficiency reasons these are usually stored as integers
-			after scaling with a scale factor. This value is the scale
-			factor. It is required to get the actual physical value,
-			when necessary.
+			An optional scaling factor to apply to the values in FIELDNAME (usually the signal).
+
+			The elements in data are often stored as integers for efficiency reasons and need
+			further correction, generating floats.  The two fields FIELDNAME_scaling_factor and
+			FIELDNAME_offset allow linear corrections using the following convention:
+
+			.. code-block::
+
+				plotted_data = (data + offset) * scaling_factor
```

Comment:
I strongly disagree with the use of "plotted_data" and "data" in this equation. Firstly, "data" is ambiguous and should be something like "stored values", "dataset values", or "recorded values". Secondly "plotted data" makes no sense because nothing is plotted (past tense) at the time when one is considering this equation and it is also meaningless because one could plot any old values. The NeXus manual says that we strive to record physically meaningful values - this equation is to be used when we do not record physically meaningful values and so its purpose should be to convert to physically meaningful values. Therefore, I would argue that "physical values" should definitely be used instead of "plotted values".